### DISCUSSION TITLE
RangeSlider fixed range

### DISCUSSION THREAD

**Discussion by vishnup-ditto** [mantinedev/mantine#2469] [Feature requests]
### What package has an issue

@mantine/core

### Describe the bug

Hey @rtivital,

How do I create a [RangeSlider](https://mantine.dev/core/slider/) with a fixed range. I see the minRange prop that's slightly useful for me. I am interested in knowing how can I implement a Rangeslider with min and max of 2. So in a scale of 1-10 values, 0-2,1-3,... will only be select able (assuming step of 1). Is this implementation possible?

### What version of @mantine/hooks page do you have in package.json?

5.2.4

### If possible, please include a link to a codesandbox with the reproduced problem

N/A

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

Need ideas here.

**Comment by rtivital** [comment]
You probably do not need range slider for that, in your case regular slider will do the job

**Reply by vishnup-ditto** [reply]
Thanks for moving this to the discussions section. Not sure I understand, the slider is for one single value right?

**Reply by rtivital** [reply]
Yes, based on your description you want single value – user cannot modify the range, only one slider can be changed. Range slider is designed to handle cases when user can change both first and second values.

**Reply by vishnup-ditto** [reply]
Maybe I was not super clear, user can modify the start,end values,[start-2,end-4],[start-3, end-5] The range value (2 here) is what remains fixed, so I am interested in 2 values.
